File: Ciência-da-Computação/INTRODUCAO-A-PYTHON/dia-1-introducao-a-python/exercise_bonus_1.py
def minimum_with_min(numbers):
    return min(numbers)


print(minimum_with_min([5, 9, 3, 19, 70, 8, 100, 2, 35, 27]))


def minimum_with_for_in(numbers):
    min = numbers[0]
    for number in numbers:
        if number < min:
            min = number
    return min


print(minimum_with_for_in([5, 9, 3, 19, 70, 8, 100, 2, 35, 27]))

File: Ciência-da-Computação/INTRODUCAO-A-PYTHON/dia-1-introducao-a-python/exercise_bonus_1.js
const minimumForEach = (numbers) => {
    let min = 100000;

    numbers.forEach((n) => n < min ? min = n : n);
    return min;

}

console.log(minimumForEach([5, 9, 3, 19, 70, 8, 100, 2, 35, 27]));

const minimumForOf = (numbers) => {
    let min = 100000;

    for(const number of numbers) {
        if(number < min) min = number;
    }
    return min;

}

console.log(minimumForOf([5, 9, 3, 19, 70, 8, 100, 2, 35, 27]));